pulp_maven/app/migrations/0007_alter_mavenartifact_unique_together_and_more.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.2.10 on 2025-02-24 20:58
+
+from django.db import migrations, models
+import django.db.models.deletion
+import pulpcore.app.util
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0126_remoteartifact_failed_at'),
```

Comment:
The reason the lowerbound test failed with 3.49 is because this migration dependency is not available on 3.49: 
```
django.db.migrations.exceptions.NodeNotFoundError: Migration maven.0007_alter_mavenartifact_unique_together_and_more dependencies reference nonexistent parent node ('core', '0126_remoteartifact_failed_at')
  ```
  ```suggestion
```